helm/ffc-workstream/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: ffc-workstream
+description: A Helm chart for Kubernetes
+type: application
+version: 1.0.0
+dependencies:
+- name: ffc-helm-library
+  version: 1.0.0
```

Comment:
Probably worth adding a caret to the version number to pick up new minor versions of the `ffc-helm-library`